### PR TITLE
ovirt-log-collector: Add --package-manager-history

### PR DESCRIPTION
**Pull request by dougsland**
Includes the yum or dnf history data per transaction.

Bug-Url: https://bugzilla.redhat.com/show_bug.cgi?id=1360621

### CODE REVIEW

**Comment by dougsland** [comment]
@didib @sandrobonazzola

File: src/__main__.py [hunk 1]
```diff
@@ -1554,21 +1555,22 @@ def cleanup():
...
+                _(
+                    'The generated archive may contain data considered sensitive '
+                    'and its content should be reviewed by the originating '
+                    'organization before being passed to any third party.'
+                ),
+                _(
+                    'No changes will be made to system configuration.\n'
+                ),
+            ):
+                print('\n'.join(textwrap.wrap(line)))
```
Comment (didib) [inline src/__main__.py]: why not logging.info as you did above?

File: src/__main__.py [hunk 2]
```diff
@@ -1098,7 +1098,8 @@ def archive(self):
         Create a single tarball with collected data from engine, postgresql
         and all hypervisors.
         """
-        print(_('Creating compressed archive...'))
+        logging.info('Creating compressed archive...')
```
Comment (didib) [inline src/__main__.py]: _()

File: src/__main__.py (hunk 1 above)
Comment (dougsland) [reply src/__main__.py]: I was not changing existing code in this patch, just adding into a if statement. We can replace with logging.info here as well in different patch.

**Comment by dougsland** [comment]
moving to gerrit.

**Pull request closed without merging**